File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/06.Nested-Loop/03-additional/08.secretDoor'sLock.js
function secretDoorsLock(input) {

    const firstNumUpperLimit = Number(input[0]);
    const secondNumUpperLimit = Number(input[1]);
    const thirdNumUpperLimit = Number(input[2]);

    for (let i = 2; i <= firstNumUpperLimit; i += 2) {
        for (let j = 2; j <= secondNumUpperLimit; j++) {
            for (let k = 2; k <= thirdNumUpperLimit; k += 2) {
                if (j == 2 || j == 3 || j == 5 || j == 7) {
                    console.log(`${i} ${j} ${k}`);
                }
            }
        }
    }
}
secretDoorsLock(["8", "2", "8"]);

File: Python_Basics/Homeworks-And-Labs/06.nested_loops/03.additional/08.secret_door's_lock.py
first_num_upper_limit = int(input())
second_num_upper_limit = int(input())
third_num_upper_limit = int(input())

for i in range(2, first_num_upper_limit + 1, 2):
    for j in range(2, second_num_upper_limit + 1):
        for k in range(2, third_num_upper_limit + 1, 2):
            if j == 2 or j == 3 or j == 5 or j == 7:
                print(f'{i} {j} {k}')
